File: fizzbuzz.js
var fizzBuzz = function (n) {
  if (n % 15 === 0) console.log("FizzBuzz");
  else if (n % 5 === 0) console.log("Buzz");
  else if (n % 3 === 0) console.log("Fizz");
  else console.log(n);
};

for(var i = 1; i <= 100; i++) {
  fizzBuzz(i)
};

File: fizzbuzz.rb
def fizzbuzz(num)
  if num % 15 == 0
    puts "Fizzbuzz"
  elsif num % 3 == 0
    puts "Fizz"
  elsif num % 5 == 0
    puts "Buzz"
  else
    puts num
  end
end

100.times do |x|
  fizzbuzz(x+1)
end
